### DISCUSSION TITLE
How to Use PKCS12 Sing

### DISCUSSION THREAD

**Discussion by projackRK** [MatthiasValvekens/pyHanko#203] [Q&A] [answered]
How to use PKCS12 for signing.

**Comment by MatthiasValvekens** [comment, marked as answer]
Judging from what you wrote in the other thread, this seems to be resolved?

Either way, for others reading this, here's a relevant code sample: https://pyhanko.readthedocs.io/en/latest/lib-guide/signing.html#creating-pades-signatures.

**Reply by projackRK** [reply]
I still can't fix this problem.

**Reply by projackRK** [reply]
pyhanko.sign.general.SigningError: Attempting to sign document with hybrid cross-reference sections while hybrid xrefs are disabled

**Reply by MatthiasValvekens** [reply]
Did you try the solution I suggested in #204?  (EDIT: the error you're seeing has pretty much nothing to do with PKCS#12, for what it's worth)
